node/core/dispute-coordinator/src/initialized.rs

```diff
@@ -869,12 +869,21 @@ impl Initialized {
 			}
 		}
 
+		let has_own_vote = new_state.has_own_vote();
+		let is_disputed = new_state.is_disputed();
+		let has_controlled_indices = env.controlled_indices().is_empty();
```

Comment:
The name of this variable seems to be at odds with the logic filling it. Should probably be opposite. !env.controlled_indices().is_empty()

Reply:
Good catch, thanks!